admin_guide/allocating_node_resources.adoc

```diff
@@ -209,22 +206,14 @@ See xref:../rest_api/index.adoc#rest-api-index[REST API Overview] for more detai
 [[node-enforcement-1]]
 == Node enforcement
 
-The node is able to limit the total amount of resources that pods
-may consume based on the configured allocatable value.  This feature significantly
-improves the reliability of the node by preventing pods from starving
-system services (for example: container runtime, node agent, etc.) for resources.
-It is strongly encouraged that administrators reserve
-resources based on the desired node utilization target
-in order to improve node reliability.
+The node is able to limit the total amount of resources that pods may consume based on the configured allocatable value.  This feature significantly improves the reliability of the node by preventing pods from starving system services (for example: container runtime, node agent, etc.) for resources. It is strongly encouraged that administrators reserve resources based on the desired node utilization target in order to improve node reliability.
```

Comment:
s/from starving system services (for example: container runtime, node agent, etc.) for resources/from depriving (?) system services, such as container runtime and node agent, of resources